--- krename-3.0.0/krename/krename.desktop.orig	Thu Mar 14 20:17:19 2002
+++ krename-3.0.0/krename/krename.desktop	Thu Jul 15 11:49:44 2004
@@ -1,5 +1,6 @@
 # KDE Config File  
 [Desktop Entry]
+Encoding=UTF-8
 Type=Application
 Exec=krename -caption "%c" %i %m  
 Icon=krename.png
@@ -7,5 +8,8 @@
 DocPath=krename/index.html
 Comment=A batch renamer
 Comment[de]=Ein Batch Umbenenner
-Terminal=0
+Comment[pl]=Zmiana nazw plików
+Terminal=false
 Name=KRename
+Categories=Qt;KDE;Utility;FileManager;
+# vi: encoding=utf-8
